tests/test_routers/test_config.py

```diff
@@ -262,9 +265,14 @@ def test_update_config(app: FastAPI, wait_for_startup_after):
     get_config = client.get("/config").json()
     assert not get_config["pipelines"]
 
+    # Empty update
+    res = client.patch("/config", json={})
+    assert res.status_code == 200
+    assert get_config == client.get("/config").json()
+
     with jsonlines.open(jsonl_file_path, "r") as reader:
         loaded_configs = list(reader)
```

Comment:
I think we are missing updating the `new_config_count`, otherwise the following `assert` doesn't test anything new.
```suggestion
        new_config_count = len(list(reader))
```

Reply:
The `new_config_count` is the variable that I set after the first patch (so equal to `initial_config_count + 1`). So here I'm checking that the length of the `loaded_configs` is still equal to `new_config_count`. Also, I need the `loaded_configs` variable for further down in the test.

Am I missing something?

Reply:
Oh I see. I was confused by the two `with ... as reader` being different, I thought it was a mistake. Sorry. That's good. 👍